pkg/ext-proc/handlers/response.go

```diff
@@ -33,3 +36,60 @@ func (s *Server) HandleResponseHeaders(reqCtx *RequestContext, req *extProcPb.Pr
 	}
 	return resp, nil
 }
+
+// HandleResponseBody parses response body to update information such as number of completion tokens.
+// Example response
+/*
+{
+    "id": "cmpl-573498d260f2423f9e42817bbba3743a",
+    "object": "text_completion",
+    "created": 1732563765,
+    "model": "meta-llama/Llama-2-7b-hf",
+    "choices": [
+        {
+            "index": 0,
+            "text": " Chronicle\nThe San Francisco Chronicle has a new book review section, and it's a good one. The reviews are short, but they're well-written and well-informed. The Chronicle's book review section is a good place to start if you're looking for a good book review.\nThe Chronicle's book review section is a good place to start if you're looking for a good book review. The Chronicle's book review section",
+            "logprobs": null,
+            "finish_reason": "length",
+            "stop_reason": null,
+            "prompt_logprobs": null
+        }
+    ],
+    "usage": {
+        "prompt_tokens": 11,
+        "total_tokens": 111,
+        "completion_tokens": 100
+    }
+}*/
+func (s *Server) HandleResponseBody(reqCtx *RequestContext, req *extProcPb.ProcessingRequest) (*extProcPb.ProcessingResponse, error) {
+	klog.V(3).Info("Processing HandleResponseBody")
+	body := req.Request.(*extProcPb.ProcessingRequest_ResponseBody)
+	klog.V(3).Infof("Response body: %+v\n", body)
```

Comment:
The body could be large, logging it for each request is a lot

Reply:
It's just for debugging purpose and  it's in V(3). I can increase the level even more if that's what you are asking.

Reply:
What log level will we recommend for prod? We need some level that allows us to effectively debug it, and so messages like line 65 will be useful, but this one should not be logged in production for sure. Same thing with line 75 and the log lines in `request.go` file.

We also need to be mindful of privacy here, and for this reason I suggest to not log full requests/responses at all.

Reply:
For production the default should be recommended (not specifying verbosity level or v=0).

That said, we do need to provide guidance on log levels. I implicitly implemented some rules but I will open a PR to update the dev guide, and a bulk update of all the log levels.

For privacy, EPP runs in user cluster so they have fully control of the logs. For now I think it's better to log more, otherwise it can be very difficult to debug.

Reply:
We need higher logging than v=0 for sure, otherwise it is hard to debug problems for running controllers.

@robscott what is the guidance on logging full response/request payload? I am not convinced that we should do that even at higher log levels.

Reply:
Removed logging the body given the concerns.

Reply:
> we do need to provide guidance on log levels

@liu-cong @ahg-g I created https://github.com/kubernetes-sigs/gateway-api-inference-extension/issues/108 to capture this requirement